### DISCUSSION TITLE
Not able to Run li in ubuntu 16.04

### DISCUSSION THREAD

**Discussion by kanagucalmet** [matt-42/lithium#68] [Q&A] [answered]
I have following,
kanagu@developer:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.7 LTS
Release:	16.04
Codename:	xenial

kanagu@developer:~$ clang++ --version
clang version 9.0.0 (https://github.com/llvm-mirror/llvm c62b24f070c9a4bb1a76409e623042a740cac4cd)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /usr/local/clang_9.0.0/bin

kanagu@developer:~$ python --version
Python 3.5.2

kanagu@developer:~$ docker --version
Docker version 18.09.7, build 2d0083d

But Still I am not able to run li using following method
$ wget https://raw.githubusercontent.com/matt-42/lithium/master/cli/li
$ chmod +x ./li

getting following error,
kanagu@developer:~$ chmod +x ./li
kanagu@developer:~$ li
li: command not found
kanagu@developer:~$ ./li run
  File "./li", line 23
    RUN cd /lithium/build && cmake .. -DCMAKE_BUILD_TYPE=Release; make install -j4;"""
                                                                                     ^
SyntaxError: invalid syntax

Please help me how can I run lithium

**Comment by matt-42** [comment, marked as answer]
Can you upgrade to python >= 3.6 ? (I just pushed a check on the python version.)

**Reply by kanagucalmet** [reply]
Its working now after updated python.
Thank you very much.
But did benchmarck using ab but its giving following error

ab -n 10 -c 10  http://127.0.0.1:8080/hello_world
This is ApacheBench, Version 2.3 <$Revision: 1706008 $>
Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
Licensed to The Apache Software Foundation, http://www.apache.org/

Benchmarking 127.0.0.1 (be patient)...apr_pollset_poll: The timeout specified has expired (70007)

but wrk works fine....
